lib/ohai/plugins/linux/network.rb

```diff
@@ -212,6 +212,31 @@ def link_statistics(iface, net_counters)
         net_counters[tmp_int] = Mash.new unless net_counters[tmp_int]
       end
 
+      if line =~ /^\s+(ip6tnl|ipip)/
+        iface[tmp_int][:tunnel_info] = {}
+        sline = line.split
```

Comment:
maybe call this `words` so that then you do `words.each_with_index do |word, index|` and then its `word[index + 1]` and that's the thing that i find easier to read than `sline[index + 1]` because at first it seems like a line, then you realize its a split line, then you think about what a split line is, then you realize its a word...

and that's a nifty trick to do lookahead...

Reply:
good call, I'll change that.

and thanks :)